### DISCUSSION TITLE
Failed to compile using the hugo server command

### DISCUSSION THREAD

**Discussion by miskin-lee** [CaiJimmy/hugo-theme-stack#963] [Q&A]
When I download the repository locally and use the template, I keep getting the following error:
Watching for changes in D:\Hugo\hello_site\{archetypes,assets,content,data,i18n,layouts,static,themes}
Watching for config changes in D:\Hugo\hello_site\config.yaml, D:\Hugo\hello_site\themes\hugo-theme-stack\config.yaml
Start building sites …
hugo v0.123.7-312735366b20d64bd61bff8627f593749f86c964+extended windows/amd64 BuildDate=2024-03-01T16:16:06Z VendorInfo=gohugoio

ERROR error calling resources.GetRemote: Get "https://publish.twitter.com/oembed?dnt=false&omit_script=true&url=https%3A%2F%2Ftwitter.com%2FDesignReviewed%2Fstatus%2F1085870671291310081": dial tcp 157.240.21.9:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
ERROR render of "home" failed: "D:\Hugo\hello_site\themes\hugo-theme-stack\layouts\index.html:9:15": execute of template failed: template: index.html:9:15: executing "main" at <partial "article-list/default" .>: error calling partial: "D:\Hugo\hello_site\themes\hugo-theme-stack\layouts\partials\article-list\default.html:3:7": execute of template failed: template: partials/article-list/default.html:3:7: executing "partials/article-list/default.html" at <partial "article/components/header" .>: error calling partial: "D:\Hugo\hello_site\themes\hugo-theme-stack\layouts\partials\article\components\header.html:34:7": execute of template failed: template: partials/article/components/header.html:34:7: executing "partials/article/components/header.html" at <partialCached "article/components/details" . .RelPermalink>: error calling partialCached: partial "article/components/details" timed out after 30s. This is most likely due to infinite recursion. If this is just a slow template, you can try to increase the 'timeout' config setting.
Built in 30099 ms
Error: error building site: render: failed to render pages: render of "page" failed: "D:\Hugo\hello_site\themes\hugo-theme-stack\layouts\_default\single.html:27:7": execute of template failed: template: _default/single.html:27:7: executing "main" at <partial "article/article.html" .>: error calling partial: partial "article/article.html" timed out after 30s. This is most likely due to infinite recursion. If this is just a slow template, you can try to increase the 'timeout' config setting.

See this kind of error, it seems to download something from one place and then time out?

**Comment by miskin-lee** [comment]
Does this happen to anyone?

**Comment by rmdhnreza** [comment]
Can your connection access to twitter.com?
Or try to delete this and run `hugo server` again.
https://github.com/CaiJimmy/hugo-theme-stack/blob/2f9306502d67377593192b1fd80b2fa0cbe32e5b/exampleSite/content/post/rich-content/index.md?plain=1#L24-L30

**Reply by miskin-lee** [reply]
Thanks for your reply, I have solved the problem. I forgot to copy something and now the blog is set up.
